### ISSUE TITLE
fwrite ISO datetime64 without Z-zone marker

### ISSUE THREAD

**Issue by avraam-inside** [Rdatatable/data.table#5440]
Hi! There is a need to fwrite datetime64 in ISO format without Z at the end of the date.

Tell me, is there a way to ask data.table to record datetime without a timezone marker?

If not, please add...

**Comment by avraam-inside** [comment]
Gentlemen, the question is quite important!

I have in the dttm table for 2022.08.29T19.08.07. In the database I have already set the timezone I am interested in. And now when I put this file to sleep, I expect to get the same 2022.08.29T19.08.07, and not 2022.08.29T23.08.07 or from another time.

This Z - is very much obstruct! And it is deleted from the file for a very long time through a linux request. It's easier to write without it right away. Is it difficult to add such a parameter (or tell how to remove Z if it is already possible)?

**Comment by MichaelChirico** [comment]
try the dateTimeAs parameter

**Comment by avraam-inside** [comment]
Yes, I need "ISO" without Z. And I don't see any settings in this parameter to exclude the letter Z from iSO!

How do I pass the condition in the parameter - do not write the letter Z at the end? (then the date will still remain in ISO format)
